programs/cleos/main.cpp

```diff
@@ -1539,9 +1539,16 @@ struct delegate_bandwidth_subcommand {
       add_standard_transaction_options(delegate_bandwidth, "from@active");
 
       delegate_bandwidth->set_callback([this] {
-          EOS_THROW(action_not_found_exception, "Action delegatebw is not supported yet");
-//         std::vector<chain::action> acts{create_delegate(from_str, receiver_str, stake, transfer)};
-//         send_actions(std::move(acts));
+          if (transfer) {
+              auto accountPermissions = get_account_permissions(tx_permission, {from_str, config::active_name});
+
+              fc::variant withdraw_params = fc::mutable_variant_object("account", from_str)("quantity", stake);
+              fc::variant transfer_params = fc::mutable_variant_object("from", from_str)("to", "cyber.stake")("quantity", stake)("memo", receiver_str);
+              send_actions({create_action(accountPermissions, N(cyber.stake), N(withdraw), withdraw_params), create_action(accountPermissions, N(cyber.token), N(transfer), transfer_params)});
+          } else {
+              std::vector<chain::action> acts{create_delegate(from_str, receiver_str, to_asset(stake), transfer)};
+              send_actions(std::move(acts));
+          }
```

Comment:
as far as I remember, in EOS, delegatebw carried out a staking. for example, to stake on your own account it was supposed to delegatebw to yourself.
so if we want to keep EOS-kind logic, we need to stake using recipient name in memo (transfer mode), or stake to self + delegateuse (not transfer mode)
although I personally prefer your version